src/domain_bridge_rti_qos.cpp

```diff
@@ -0,0 +1,253 @@
+// Copyright 2021, Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <sstream>
+#include <string>
+
+#include "rclcpp/executors/single_threaded_executor.hpp"
+#include "rcpputils/scope_exit.hpp"
+#include "rcutils/cmdline_parser.h"
+#include "rcutils/env.h"
+
+#include "domain_bridge/domain_bridge.hpp"
+#include "domain_bridge/parse_domain_bridge_yaml_config.hpp"
+
+
+#ifndef _WIN32
+# pragma GCC diagnostic push
+# pragma GCC diagnostic ignored "-Wpedantic"
+# pragma GCC diagnostic ignored "-Wunused-parameter"
+# ifdef __clang__
+#  pragma clang diagnostic ignored "-Wdeprecated-register"
+#  pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
+# endif
+#endif
+#include "ndds/ndds_c.h"
+// #include <ndds/ndds_namespace_cpp.h>
+// #include <ndds/ndds_requestreply_cpp.h>
```

Comment:
I guess we can remove these unused includes

Reply:
Removed https://github.com/ros2/domain_bridge/pull/25/commits/772d374de4e7d4742c67ea1218a906d027f11c2d